src/Directory.Build.props

```diff
@@ -48,6 +48,12 @@ Source code and extensive sample application available at http://www.hardcodet.n
         <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
       </PropertyGroup>
 
+      <PropertyGroup Condition="$(TargetFramework.StartsWith('net4'))">
```

Comment:
Why only for net4xxx?

Reply:
I believe that strong naming a dotnet core assembly doesn't add any value.
But I'm not 100% sure, now reading through all the available information seems to tell me that dotnet core assemblies are string named.

What is your opinion?